src/main/java/ticTacToe/game/Mark.java

```diff
@@ -9,4 +9,11 @@
    }
 
     public String mark;
+
+    public static String switchPlayerMark(String currentMark) {
```

Comment:
How come you are passing a `String` in here? Couldn't you pass in the `Mark` itself?
No need to use the Strings `"O"` and `"X"` for anything other than printing/displaying the board.

Reply:
These Enums... have to familiarise better with them

Reply:
I made these changes on my ui-improvements branch (next PR)